lib/govuk_publishing_components/presenters/component_wrapper_helper.rb

```diff
@@ -192,6 +199,15 @@ def check_dir_is_valid(dir_attribute)
         end
       end
 
+      def check_type_is_valid(type_attribute)
+        return if type_attribute.nil?
+
+        options = %w[button checkbox file hidden radio reset submit text]
```

Comment:
I think more types need to be included - unless you're only including ones we currently use? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types

Reply:
Initially I only included the ones for the button component, since we don't need any others currently, but I added some more after I went through the list based on what we might probably need in the near future.

I think since the component wrapper helper is normally used on the outside element of a component it's not likely to be used for actual inputs, so we probably don't need radio/checkbox/etc. I'm not sure. Shall I just include the ones needed for this change and add any others later? So far the button component is the only component that needs `type`.

Reply:
That makes sense, I guess my concern was that `aria`, `lang` etc. seem to have all the valid values included - for example we have the latin language code `la` but I suspect we won't have any Latin components on GOVUK! So would things go a bit out of sync if all the others contain the valid ?

Reply:
That's reasonable. I'll add the others.